scripts/Units/armcroc.bos

```diff
@@ -130,6 +148,69 @@ AimFromPrimary(piecenum)
 	piecenum = turret;
 }
 
+
+
+
+
+
+
+
+
+AimSecondary(heading, pitch)
+{
+	signal SIG_AIM2;
+	set-signal-mask SIG_AIM2;
+	
+	move aturret to y-axis [15] speed [100];
+	move launcher to y-axis [4.5] speed [100];
+	turn launcher to x-axis <90.000000> speed <200.000000>;
+	
+	if( gun_2 == 1 )
+	{
+		turn asleeve to y-axis <0.000000> speed <400.000000>;
+	}
+	if( gun_2 == 2 )
+	{
+		turn asleeve to y-axis <120.027473> speed <400.000000>;
+	}
+	if( gun_2 == 3 )
+	{
+		turn asleeve to y-axis <-120.027473> speed <400.000000>;
+	}
+	turn aturret to y-axis <180> + heading speed <450.000000>;
+	total = <90.000000> + pitch;
+	if (total < 20000) {
+		total = <120>;
```

Comment:
I assume the above means the 20000 comes from some reliable empirical measurement. Why not use 20000 instead of <120> here then?

Reply:
Last time I tried using <x> in a comparator it refused to compile, so I'd thought it was impossible to do that way - seems to work though, so possibly it was something else? Will test in a bit and check it works in game, but using angles is certainly cleaner if it does in fact behave.